core/src/main/java/org/mule/runtime/core/api/config/MuleDeploymentProperties.java

```diff
@@ -14,9 +14,6 @@
   public static final String DEPLOYMENT_PROPERTY_PREFIX = "mule.application.deployment";
   public static final String MULE_MUTE_APP_LOGS_DEPLOYMENT_PROPERTY = DEPLOYMENT_PROPERTY_PREFIX + ".muteLog";
 
-  public static final String MULE_DISABLE_TEST_CONNECTION_ON_DEPLOYMENT_PROPERTY =
```

Comment:
This is API, I would not remove it

Reply:
I have added that in a previous commit, so, it was not released yet. It has been part of a timestamp based build but not yet released. Those are internal "releases" that we have been doing for validating the changes on master.

Reply:
Ok, I did not know that.